src/client/Microsoft.Identity.Client/ManagedIdentity/KeyProviders/KeyGuardHelper.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Security.Cryptography;
+using Microsoft.Identity.Client.Core;
+
+namespace Microsoft.Identity.Client.ManagedIdentity.KeyGuard
+{
+    /// <summary>
+    /// Helper that creates or opens a Key Guard–protected RSA key.
+    /// </summary>
+    internal static class KeyGuardHelper
+    {
+        private const string ProviderName = "Microsoft Software Key Storage Provider";
+        private const string KeyName = "KeyGuardRSAKey";
```

Comment:
You're using these constants in more than one file? Can you extract them to a shared Constants file?